main.tf

```diff
@@ -84,8 +84,9 @@ data "lacework_user_profile" "current" {}
 
 data "google_client_config" "default" {}
 
+// if the scanning project id is not provided, use the project specified in the provider
 data "google_project" "selected" {
-  count = length(var.scanning_project_id) > 0 ? (length(var.organization_id) > 0 ? 0 : 1) : 1
+  count = length(var.scanning_project_id) > 0 ? 0 : 1
```

Comment:
I think this will make the logic here less convoluted. Previously there's an implicit dependency:

* `local.scanning_project_id` depends on `data.google_project.selected` depends on `var.organization_id`

which made things a bit hard to reason about. Hence I'm doing this cleanup, so we'll always try to get the current project regardless whether `organization_id` is provided. LMK if this condition is actually needed.